client/app/pages/alert/AlertView.jsx

```diff
@@ -46,6 +45,7 @@ AlertState.defaultProps = {
   lastTriggered: null,
 };
 
+// eslint-disable-next-line react/prefer-stateless-function
 export default class AlertView extends React.Component {
```

Comment:
Seems it could be a functional component (`this` is not used anywhere)

Reply:
Oh forgot to mention, I'm adding state to this component in #4276  so no point in changing it.

https://github.com/getredash/redash/blob/c01fbf90619d9053623690892c54bac827651089/client/app/pages/alert/AlertView.jsx#L51-L60